invenio_rdm_records/marshmallow/json.py

```diff
@@ -81,8 +82,19 @@ class AffiliationSchemaV1(BaseSchema):
     """Affiliation of a creator/contributor."""
 
     name = SanitizedUnicode(required=True)
-    identifier = SanitizedUnicode(required=True)
-    scheme = SanitizedUnicode(required=True)
+    identifiers = fields.Dict()
+
+    @validates("identifiers")
+    def validate_identifiers(self, value):
+        """Validate well-formed identifiers are passed."""
+        if len(value) == 0:
+            raise ValidationError(_("Invalid identifier."))
```

Comment:
@fenekku raising errors without specifying the field that they refer to, it causes the issue we have in the deposit for not being able to connect an error with the field. Do you think we could pass the field here as per any other default Marshmallow validation error? I am ok if you want to leave it like this and fix all together later but I just see that they are pilling up :) What do you think?

Reply:
> raising errors without specifying the field that they refer to, it causes the issue we have in the deposit for not being able to connect an error with the field. 

Ah thanks for alerting me to that. Since I am looking into errors for flask-resources, I can look into errors for this too after. I made an issue for it: https://github.com/inveniosoftware/invenio-rdm-records/issues/127 since I think it will take more time to wrap my head around a good way to deal with this.

Reply:
@fenekku inveniosoftware/react-invenio-deposit#22 is not only for styling as the errors are not displaying at all if the error message is not referring to a specific field :)